src/main/java/com/microsoft/azure/functions/worker/JavaWorkerClient.java

```diff
@@ -22,7 +22,7 @@
 public class JavaWorkerClient implements AutoCloseable {
     public JavaWorkerClient(IApplication app) {
         WorkerLogManager.initialize(this, app.logToConsole());
-        ManagedChannelBuilder<?> chanBuilder = ManagedChannelBuilder.forAddress(app.getHost(), app.getPort()).usePlaintext(true);
+        ManagedChannelBuilder<?> chanBuilder = ManagedChannelBuilder.forAddress(app.getHost(), app.getPort()).usePlaintext();
```

Comment:
Could you share why you changed this part?

Reply:
usePlaintext(boolean skipNegotiation) is currently deprecated. Hence I updated it accordingly.
https://javadoc.io/static/io.grpc/grpc-core/1.20.0/io/grpc/ManagedChannelBuilder.html

Reply:
That's make sense.